src/analytics/analytics.ts

```diff
@@ -136,3 +136,33 @@ export interface SubscriberMetrics {
   total_subscribers: number
   date: string
 }
+
+export interface RevenueMetricsValue {
+  label: string
+  distinct_person_count: number
+  conversation_marketing_count: number
+  conversation_utility_count: number
+  journeys_count: number
+  journeys_revenue_gross_total: number
+  journeys_revenue_net_total: number
+  campaigns_count: number
+  campaigns_revenue_net_total: number
+  campaigns_revenue_gross_total: number
+  attribution_event_count: number
+  per_conversation_mean: number
+  revenue_gross_total: number
+  revenue_net_total: number
```

Comment:
little update here: These can be optional since the `fields` query param can control which field you want. And perhaps split these out so that we can use it separately as `keyof`

additionally, there is now a `conversation_business_count` that is the sum of the two other fields